src/mjolnir/pbfgraphparser.cc

```diff
@@ -1631,17 +1631,15 @@ OSMData PBFGraphParser::Parse(const boost::property_tree::ptree& pt,
 
   // we need to sort the from complex restrictions so that we can easily find them.
   LOG_INFO("Sorting complex restrictions by from id...");
-  {
```

Comment:
why remove the scopes? they were there to make sure that the files are closed when we are done accessing them. it may not be necessary but imho it is good practice to not let file handles open for no reason for extended periods of time.